### PR TITLE
Streamline error handling

### PR DESCRIPTION
**Pull request by rudolfs**
Streamline our 404 not found pages by showing the name of the resource that wasn't found, e.g. `Page`, `Issue`, `Patch`, `Commit` and `Seed`.

We remove the `Back` button which simplifies the router logic, because we don't have to keep track of the browser history manually. Users can use the browser back button.

Finally we add tests for the 404 and loading error pages.

In a follow-up we'll implement a global unhandled exception page, which could possibly replace the `loadError` route.

Refs: https://github.com/radicle-dev/radicle-interface/issues/986.
Closes: https://github.com/radicle-dev/radicle-interface/issues/894.

### CODE REVIEW

File: tests/support/peerManager.ts [hunk 1]
```diff
@@ -287,24 +293,23 @@ export class RadiclePeer {
...
-                const result: NodeEvent = JSON.parse(data);
-                this.#eventRecords.push(result);
-              } catch (e) {
-                console.log("Error parsing event", data);
-              }
-            }
-          });
-      },
-    );
+    const stdout = this.rad(["node", "events"], { cwd: this.#radHome }).stdout;
```
Comment (sebastinez) [inline tests/support/peerManager.ts:296]: Really nice improvement! thanks 🙇

**Review by sebastinez — APPROVED** [review]
Okay tested the code paths and agree with the simplicity of the NotFound view 👍

File: tests/visual/landingPage.spec.ts [hunk 2]
```diff
@@ -19,3 +19,14 @@ test("landing page", async ({ page }) => {
   await page.goto("/", { waitUntil: "networkidle" });
   await expect(page).toHaveScreenshot();
 });
+
+test("load error", async ({ page }) => {
```
Comment (sebastinez) [inline tests/visual/landingPage.spec.ts:23]: I think we should add the FakeTimers to this test, else the activity is going to be shifting every week or so.

**Review by sebastinez — COMMENTED** [review]
Just some minor things I saw

File: tests/visual/landingPage.spec.ts (hunk 2 above)
Comment (rudolfs) [reply tests/visual/landingPage.spec.ts:23]: Good catch. Fixed.

**Review by sebastinez — APPROVED** [review]
Nice, LGTM

**Pull request merged**